controllers/octavia_controller.go

```diff
@@ -821,6 +821,7 @@ func (r *OctaviaReconciler) amphoraControllerDaemonSetCreateOrUpdate(
 		daemonset.Spec.ServiceAccount = instance.RbacResourceName()
 		daemonset.Spec.LbMgmtNetworks.ManageLbMgmtNetworks = instance.Spec.LbMgmtNetworks.ManageLbMgmtNetworks
 		daemonset.Spec.LbMgmtNetworks.SubnetIPVersion = instance.Spec.LbMgmtNetworks.SubnetIPVersion
+		daemonset.Spec.LoadBalancerCerts = instance.Spec.OctaviaHousekeeping.LoadBalancerCerts
```

Comment:
This works, but it seems wrong to me. `LoadBalancerCerts` is part of `OctaviaAmphoraControllerSpec`, and I think that makes sense because it is something amphora specific. However, AFAIU that makes it necessary to set a value for this for the all the worker, health-manager and housekeeping services in the YAML, which is redundant information and error prone to sync.

The underlying issue is that the default value does not get applied, and that seems to be a known issue with kubebuilder and not easy to fix (essentially the defaults would need to be define in the parent `OctaviaSpec` AFAIU). So, for now one has to manually edit `openstackcontrolplanes` and set the `octaviaHousekeeping.certssecret` value to "octavia-certs-secret".